ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/forktransition/ElectraStateUpgrade.java

```diff
@@ -153,11 +152,10 @@ public BeaconStateElectra upgrade(final BeaconState preState) {
   }
 
   private UInt64 findEarliestExitEpoch(final BeaconState state) {
-    final SszList<Validator> validators = state.getValidators();
     UInt64 lastExitEpoch = UInt64.ZERO;
-    for (int i = 0; i < validators.size(); i++) {
-      final UInt64 exitEpoch = validators.get(i).getExitEpoch();
-      if (exitEpoch.isLessThan(UInt64.MAX_VALUE)) {
+    for (final Validator validator : state.getValidators()) {
+      final UInt64 exitEpoch = validator.getExitEpoch();
+      if (!exitEpoch.equals(FAR_FUTURE_EPOCH)) {
         lastExitEpoch = lastExitEpoch.max(exitEpoch);
       }
```

Comment:
Actually, I think I just noticed a problem in `findEarliestExitEpoch`. If none of the validators have an exit epoch, `lastExitEpoch` should be the current epoch, not zero.